packages/remix-cloudflare/index.ts

```diff
@@ -32,7 +32,6 @@ export type {
   CookieParseOptions,
   CookieSerializeOptions,
   CookieSignatureOptions,
-  CreateRequestHandlerFunction,
```

Comment:
`CreateRequestHandlerFunction` is a utility for authoring Remix runtime packages and was incorrectly re-exported before.